other/upgrade.php

```diff
@@ -28,15 +35,6 @@
 		'utf8_version_check' => 'global $db_connection; return mysqli_get_server_info($db_connection);',
 		'alter_support' => true,
 	),
-	'mysql' => array(
-		'name' => 'MySQL',
-		'version' => '5.0.3',
-		'version_check' => 'return min(mysql_get_server_info(), mysql_get_client_info());',
-		'utf8_support' => true,
-		'utf8_version' => '5.0.3',
-		'utf8_version_check' => 'return mysql_get_server_info();',
-		'alter_support' => true,
-	),
 	'postgresql' => array(
 		'name' => 'PostgreSQL',
 		'version' => '8.0',
```

Comment:
From this comment here; http://www.simplemachines.org/community/index.php?topic=547286.msg3890058#msg3890058

PostgreSQL is meant to be version 9.1 minimum now.